src/content/blog/2023_intern_Srila.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Meet the UF interns - Srila Palanikumar
+author: Srila Palanikumar
+day: 11
+month: 8
+year: 2023
+---
+
+As part of Qubit by Qubit’s Early Quantum Career Immersion program, I was placed as a summer 2023 intern with Unitary Fund. This program was created to give students who are underrepresented in quantum computing opportunities to advance their careers, as well as to amplify their voices and support them. The first three weeks of this program were rigorous training in concepts in quantum computing and mechanics, and the next seven were a full-time internship with UF.
+
+## About me
+
+I’m a third-year student at UT Austin studying computational biology and mathematics. I’m also pursuing a minor in computer science. I’m interested in biophysics and quantum chemistry, and I enjoy working out and reading in my free time. At UT, I’ve worked on research projects involving ecological data mining and QML for linear algebra.
+
+## My internship
+
+The internship started on June 26th, after the training period with QxQ. The first big task I was assigned by my primary supervisors, Dan Strano and Vincent Russo, was to get the development environment set up locally on my computer. This required me to understand a lot of the basics of software development that I had never learned in my classes. For example, I started using virtual environments and managing packages. Around the same time, I was assigned to add research papers to [Metriq](https://metriq.info/). This helped me learn a lot of basic concepts in quantum computing, since I had to parse through the papers to determine their findings.
```

Comment:
```suggestion
The internship started on June 26th, 2023, after the training period with QxQ. The first big task I was assigned by my primary supervisors, Dan Strano and Vincent Russo, was to get the development environment set up locally on my computer. This required me to understand a lot of the basics of software development that I had never learned in my classes. For example, I started using virtual environments and managing packages. Around the same time, I was assigned to add research papers to [Metriq](https://metriq.info/). This helped me learn a lot of basic concepts in quantum computing, since I had to parse through the papers to determine their findings.
```